CHANGELOG.md

```diff
@@ -1,3 +1,292 @@
+- Makefile: use latest Go when building in Docker
+  ([#1083](https://github.com/kinvolk/lokomotive/pull/1083)).
+- cli/cmd: cleanups part 3
+  ([#1018](https://github.com/kinvolk/lokomotive/pull/1018)).
+- Add new CI config for Packet based FLUO testing
+  ([#1110](https://github.com/kinvolk/lokomotive/pull/1110)).
+
+### Updating from v0.4.1
+
+#### Configuration syntax changes
+
+There have been some minor changes to the configurations of worker nodes. The data type of `labels` and `taints` has been changed from `string` to `map(string)` for the AWS and Packet platforms.
+
+##### Old:
+
+```sh
+labels = "testing=true"
+
+taints = "nodeType=storage:NoSchedule"
+```
+
+##### New:
+
+```sh
+labels = {
+  "testing" = "true"
+}
+
+taints = {
+  "nodeType" = "storage:NoSchedule"
+}
+```
+
+This release also changes the default `cluster.oidc.client_id` value from `gangway` to `clusterauth`. This setting must match `gangway.client_id` and `dex.static_client.id`. If you use default settings for oidc you'll need to add `client_id = "gangway"` or change the `static_client.id` and `client_id` parameters for dex and gangway to `clusterauth` respectively.
+
+##### Old:
+
+```sh
+packet {
+  oidc {
+    client_id = "gangway"
+  }
+}
+```
+
+##### New:
+
+```sh
+packet {
+  oidc {
+    client_id = "clusterauth"
+  }
+}
+```
+
+#### Cluster update steps
+
+Due to [Terraform](https://github.com/kinvolk/lokomotive/pull/824) and [Kubernetes](https://github.com/kinvolk/lokomotive/pull/1030) updates to v0.13+ and v1.19.3 respectively, some manual steps need to be performed when updating. In your cluster configuration directory, follow these steps:
+
+1. Update local Terraform binary to version v0.13.X. You can follow [this guide](https://learn.hashicorp.com/tutorials/terraform/install-cli.) to do that.
+
+2. Starting from your cluster directory, export your platform name and assets directory name used in your platform configuration. It will be used in next steps:
+  ```sh
+  export PLATFORM="packet" && export ASSETS_DIR="assets"
+  ```
+
+3. Remove old asset files:
+  ```sh
+  rm $ASSETS_DIR/terraform-modules/$PLATFORM/flatcar-linux/kubernetes/require.tf \
+  $ASSETS_DIR/terraform-modules/$PLATFORM/flatcar-linux/kubernetes/workers/require.tf \
+  $ASSETS_DIR/terraform-modules/dns/route53/require.tf 2> /dev/null
+  ```
+
+4. Go to the `terraform` directory:
+  ```sh
+  cd $ASSETS_DIR/terraform
+  ```
+
+5. Replace the old providers:
+  ```sh
+  terraform state replace-provider -auto-approve registry.terraform.io/-/ct registry.terraform.io/poseidon/ct && \
+  terraform state replace-provider -auto-approve registry.terraform.io/-/template registry.terraform.io/hashicorp/template
+  ```
+
+6. **NOTE:** On clusters with a single controller node only, delete the old `kube-apiserver` ReplicaSet:
+  ```sh
+  kubectl delete rs -n kube-system $(kubectl get rs -n kube-system -l k8s-app=kube-apiserver --no-headers=true  | sort -k5 -h | head -n +2 | awk '{print $1}')
```

Comment:
This must be executed during cluster upgrade, so when `lokoctl cluster apply` writes that it's updating `kube-apiserver` system component.

Reply:
should it include it after apply or write the instruction here?